code/modules/reagents/chemistry/holder.dm

```diff
@@ -161,6 +161,14 @@
 		if(preserve_data)
 			trans_data = copy_data(T)
 		R.add_reagent(T.id, transfer_amount * multiplier, trans_data, chem_temp, no_react = 1) //we only handle reaction after every reagent has been transfered.
+		if(istype(T, /datum/reagent/consumable/ethanol/customizable))
+			var/datum/reagent/consumable/ethanol/customizable/ale = R.has_reagent("customizable_ale")
```

Comment:
You don't use the `ale` variable anywhere else, why not just put the `R.has_reagent` in the `if` statement? Makes it a bit more readable in my opinion.